orb/core/column_types/data.py

```diff
@@ -38,7 +38,7 @@ def dbRestore(self, db_value, context=None):
         """
         if db_value is not None:
             try:
-                return pickle.loads(db_value)
+                return pickle.loads(str(db_value))
```

Comment:
a cPickle is a kind of python data construct, one of the options for saving/loading DB data.  we don't use it...but it's available.  apparently pickle.loads requires a str